gcloud/connection.py

```diff
@@ -305,12 +308,12 @@ def api_request(self, method, path, query_params=None,
                             latest API version supported by
                             gcloud-python.
 
-        :type expect_json: boolean
+        :type expect_json: bool
         :param expect_json: If True, this method will try to parse the
                             response as JSON and raise an exception if
                             that cannot be done.  Default is True.
 
-        :type _target_object: object or :class:`NoneType`
+        :type _target_object: :class:`object` or :class:`NoneType`
```

Comment:
Odd that `dict` and `bool` are understood, but not `object` (and even `None`/`NoneType`).